code/game/objects/obj_defense.dm

```diff
@@ -134,7 +134,6 @@
 /obj/attack_slime(mob/living/simple_animal/slime/user)
 	if(!user.is_adult)
 		return
```

Comment:
this whole block has to go

Reply:
If I delete the block isn't that going to cause an issue with every other time attack_slime is called?

Reply:
Oh wait, I'm misreading all of this.

Reply:
if you don't see `proc` in the line that means it'll inherit from somewhere (in this case atoms) if you delete it.

the other attack_slimes will not change unless they called inheritance, in which case they will be affected by any change here.